lib/nano.js

```diff
@@ -251,128 +238,119 @@ module.exports = exports = function dbScope (cfg) {
 
     log({ err: 'couch', body: message, headers: responseHeaders })
 
-    stream.emit('error', error)
+    setTimeout(() => {
+      stream.emit('error', error)
+    }, 10)
   }
 
-  function relax (opts, callback) {
+  function relax (opts) {
     if (typeof opts === 'function') {
-      callback = opts
       opts = { path: '' }
     }
-
     if (typeof opts === 'string') {
       opts = { path: opts }
     }
-
     if (!opts) {
       opts = { path: '' }
-      callback = null
     }
 
-    const qs = Object.assign({}, opts.qs)
+    // the building blocks of the request
+    let body, uri
 
+    // construct headers object to be passed in the request
     const headers = {
-      'content-type': 'application/json',
+      // 'content-type': 'application/json',
```

Comment:
why add new commented-out code here?

Reply:
fixed in d8b5d67c2713eda6b0dfcb7b6c0fc96f5fff167f